### ISSUE TITLE
Import of Java Methods

### ISSUE THREAD

**Issue by magnus-madsen** [flix/flix#609]
We currently support three verbose constructs for interop with Java:

```
native new foo.bar.Baz();
native method foo.Bar.baz(x);
native field foo.Bar.baz(x);
```

We can use these to create new objects, invoke methods, and read fields. 
We have no mechanism to write fields (nor to read/write static fields?)
Moreover, we can refer to Java types with `##`, e.g. `##java.lang.String`.

In some cases, due to overloading, it is necessary to add type ascriptions, e.g.:

```
native method foo.Bar.baz(x: Int, y: Str);
```

This scheme works OK, but it quickly gets tedious to write out the full names and types.

A better approach might be to allow Java types and constructs to be imported with Flix names.

We could import types:

```scala
import jvm class java.lang.Object as JObject
import jvm class java.nio.Files.Path as JPath
```

Note: The names of the imported types must be uppercase. 

And import constructors:

```scala
import jvm constructor java.lang.Object() as newObject
import jvm constructor java.lang.String() as newString
import jvm constructor java.lang.String(Array[Int8]) as newStringFromByteArray
```

Note: The names of imported methods must be  lowercase.
Note: We can specify the argument types to resolve overloaded constructors.
Note: We can refer to imported types.

And import methods:

```scala
import jvm method java.lang.Object.toString(JObject) as toString
import jvm method java.nio.files.Path.endsWith(JPath, JPath) as endsWith
```

Note: Approximately similar to importing constructors.

And import fields:

```scala
import jvm getfield com.mycompany.Person.name as getName
import jvm setfield com.mycompany.Person.name as setName
```

Questions:
- How should the return type of a method be annotated? 
- Do we want to allow the signatures to contain the parameter names?
- How confusing is it that the first argument is the implicit this argument? E.g. when importing toString.

NB: The keyword `use` might be more appropriate than `import`.

**Comment by magnus-madsen** [comment]
@FkirkK Do you have any thoughts on the syntax or if any functionality might be missing?

**Comment by magnus-madsen** [comment]
Implemented in #939.
